src/utils/generate-configuration.ts

```diff
@@ -1,75 +1,44 @@
 import { Value } from "@sinclair/typebox/value";
 import { DefinedError } from "ajv";
-import merge from "lodash/merge";
+import mergeWith from "lodash/merge";
 import { Context as ProbotContext } from "probot";
 import YAML from "yaml";
 import Runtime from "../bindings/bot-runtime";
 import { BotConfig, Payload, stringDuration, validateBotConfig } from "../types";
-import ubiquibotConfigDefault from "../ubiquibot-config-default";
 
 const UBIQUIBOT_CONFIG_REPOSITORY = "ubiquibot-config";
 const UBIQUIBOT_CONFIG_FULL_PATH = ".github/ubiquibot-config.yml";
 
 export async function generateConfiguration(context: ProbotContext): Promise<BotConfig> {
   const payload = context.payload as Payload;
 
-  const organizationConfiguration = parseYaml(
+  const orgConfig = parseYaml(
     await download({
       context,
       repository: UBIQUIBOT_CONFIG_REPOSITORY,
       owner: payload.organization?.login || payload.repository.owner.login,
     })
   );
 
-  const repositoryConfiguration = parseYaml(
+  const repoConfig = parseYaml(
```

Comment:
I prefer that you do not shorten words. The reason why is to reduce cognitive overhead when reading through all of the code.